labs/SettingsControls/src/SettingsCard/SettingsCard.xaml

```diff
@@ -956,12 +952,7 @@
                                               Visibility="Collapsed" />
                             <Grid Grid.Row="1"
                                   HorizontalAlignment="Right"
-                                  VerticalAlignment="Top">
-                                <Grid.RowDefinitions>
-                                    <RowDefinition Height="{ThemeResource ToggleSwitchPreContentMargin}" />
```

Comment:
Is this resource used anymore then?

Reply:
Nope.. but that's in the WinUI resources file.